test/__tests__/data/fb_custom_audience_router.json

```diff
@@ -169,150 +169,115 @@
     ],
     "output": [
       {
-        "batchedRequest": {
```

Comment:
Are we not having batched multiplexed testcases for iterable and snapchat conversion?

`non-blocking`

Reply:
It was always there, I have updated according to new format.